build.fsx

```diff
@@ -254,7 +259,7 @@ open Fake.Core.TargetOperators
   ==> "UpdateSpec"
   ==> "UpdateBindings"
   ==> "Build"
-  ==> "CopyBinaries"
+//  ==> "CopyBinaries" // Disabled for now, it isn't necessary and doesn't work.
```

Comment:
Instead of disabling with // wouldnt it be better to disable by removing the line entirely?